README.md

```diff
@@ -91,11 +91,11 @@ visit [https://osquery.io/downloads](https://osquery.io/downloads/).
 
 ## Build from source
 
-Building osquery from source is encouraged! Check out our [build guide](https://osquery.readthedocs.io/en/latest/development/building/). Also check out our [contributing guide](CONTRIBUTING.md) and join the community on [Slack](https://slack.osquery.io).
+Building osquery from the source is encouraged! Check out our [build guide](https://osquery.readthedocs.io/en/latest/development/building/). Also, check out our [contributing guide](CONTRIBUTING.md) and join the community on [Slack](https://slack.osquery.io).
```

Comment:
I'd push back on adding `the`. I _definitely_ could be wrong as I'm not amazing at english, but I think one might more commonly say `build the project from source`, as opposed to `build the project from the source`.

Reply:
Further I'm not totally convinced we need the comma you're adding after `Also`.